pyth-sdk-terra/README.md

```diff
@@ -0,0 +1,71 @@
+# Pyth SDK Terra
+
+This crate provides utilities for reading price feeds from the [pyth.network](https://pyth.network/) oracle on the Terra network.
+The crate includes a library for reading and using Pyth data feeds in Terra.
+
+## Usage
+
+### Read price
+ 
+For reading the price you just need to call `query_price` function within your contract.
+
+```rust
+// let price_id = Binary::from(b"xyz...");
```

Comment:
How do users get this ID? I think you need to point them to the pyth.network website and explain that this is a solana account key.

Reply:
Yes I agree.

Reply:
Instead I added the list here for two reasons.
1. Pyth website is very complicated and it's hard for people not familiar with Solana to understand what is the id.
2. Not all the prices are available. 

Eventually we can move these to our gitbook.